.travis.yml

```
@@ -1,9 +1,9 @@
 language: php
 
 php:
-  - 7.2
   - 7.3
   - 7.4
+  - 8.0
```

```suggestion
  - 8.0snapshot
```

`8.0` doesn't exist in Travis yet.